src/api/app/helpers/webui/request_helper.rb

```diff
@@ -194,4 +194,10 @@ def request_action_header(action, creator)
     end.html_safe
   end
   # rubocop:enable Style/FormatStringToken, Style/FormatString
+
+  def review_request_reason(bs_request, review)
+    reason = bs_request.request_history_elements.where(description_extension: review[:id]).pluck(:comment).first
+
+    reason.presence ? reason : 'No reason given'
```

Comment:
It can be reduced to `reason.presence || 'No reason given'`

Reply:
You are right. Nice!